stream-chat-android-pushprovider-xiaomi/build.gradle

```diff
@@ -0,0 +1,84 @@
+import io.getstream.chat.android.Dependencies
+import io.getstream.chat.android.Configuration
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'org.jlleitschuh.gradle.ktlint'
+apply plugin: "de.mannodermaus.android-junit5"
+
+ext {
+    PUBLISH_GROUP_ID = Configuration.artifactGroup
+    PUBLISH_ARTIFACT_ID = 'stream-chat-android-pushprovider-xiaomi'
+    PUBLISH_VERSION = rootVersionName
+}
+
+apply from: "${rootDir}/scripts/publish-module.gradle"
+apply from: "${rootDir}/scripts/android.gradle"
+
+
+android {
+    defaultConfig {
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        buildConfigField 'String', 'STREAM_CHAT_VERSION', "\"$version\""
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            consumerProguardFiles 'consumer-proguard-rules.pro'
+        }
+        debug {
+            testCoverageEnabled true
+            consumerProguardFiles 'consumer-proguard-rules.pro'
+        }
+    }
+
+    resourcePrefix 'stream_'
+
+    lintOptions {
+        xmlReport false
+    }
+
+    testOptions.unitTests {
+        returnDefaultValues = true
+    }
+}
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        freeCompilerArgs += [
+                '-progressive',
+                '-Xexplicit-api=strict',
+                '-Xopt-in=io.getstream.chat.android.core.internal.InternalStreamChatApi',
+        ]
+    }
+}
+
+dependencies {
+    api project(':stream-chat-android-client')
```

Comment:
Nobody will add this push provider as their only dependency, so this could be an `implementation` dependency here